src/scss/base.scss

```diff
@@ -189,6 +194,78 @@
 		@include oFormsGroupInlineRadioCheckbox($class);
 	}
 }
+/// @access public
+/// @param {string} $class - The block level class name (BEM naming convention).
+/// @output Styles for form status such as "saving" and "saved".
+@mixin oFormsStatusFeature($class: 'o-forms') {
+	// Icons align to a 40x40 pixel grid with 10px padding on each side.
+	// In this case however we want to remove that padding to align the icon flush with the input.
+	// https://github.com/Financial-Times/fticons/blob/master/contributing.md#design
+	$status-icon-size: $_o-forms-field-small-height;
+	$status-icon-padding: ($status-icon-size/40) * 10;
+	@include oBrandConfigureFor($component: 'o-forms') {
+		// Base status styles.
+		.#{$class}__status {
+			@include oTypographySize($_o-forms-typography-scale-small);
+			opacity: 0;
+			visibility: hidden;
+			display: flex;
+			white-space: nowrap;
+			align-items: center;
+			min-width: 5em;
+			&:after {
+				content: '';
+			}
+			&:before {
+				content: '';
+				margin: #{-$status-icon-padding} 0 #{-$status-icon-padding} #{-$status-icon-padding};
+			}
+		}
+
+		.#{$class}__status[aria-hidden="false"] {
+			display: flex;
+			visibility: visible;
+			opacity: 1;
+		}
+
+		// Saving status.
+		.#{$class}--saving .#{$class}__status {
+			&:after {
+				content: 'Saving';
```

Comment:
Just to be clear — the request for this feature is explicitly for a 'Saving' scenario? Would it need to handle different wording in the future?

Reply:
Different status may be introduced but I think that can be handled in this way too. We don't currently have a need for dynamic or translated content.

Reply:
👍